GVFS/GVFS.UnitTests/Common/ModifiedPathsDatabaseTests.cs

```diff
@@ -17,6 +18,17 @@ public class ModifiedPathsDatabaseTests
         private const string ExistingEntries = @"A file.txt
 A dir/file2.txt
 A dir1/dir2/file3.txt
+";
+        private const string EntriesToCompress = @"A file.txt

```

Comment:
I think we need to test more data shapes than this. Adds and deletes, for one thing, but also things like both a file and a folder with the same name, and perhaps others as well.

This feature feels like it carries some risk, so we definitely need to make sure we're getting enough test coverage on it. Thoughts on adding some functional tests as well?

Reply:
Sure we already have the `ModifiedPathsTests` that we can add to.